src/components/ha-camera-stream.ts

```diff
@@ -61,6 +64,7 @@ class HaCameraStream extends LitElement {
             <ha-hls-player
               autoplay
               playsinline
+              ?allowexoplayer=${this.allowExoPlayer}
```

Comment:
```suggestion
              .allowExoplayer=${this.allowExoPlayer}
```

Reply:
I thought the `.allowExoplayer` way would work but it didn't seem to work when I tried it before. The current way seems to work. I can try again, maybe there was another problem.